README.md

```diff
@@ -2,58 +2,206 @@
+Required Parameters: None
+Optional Parameters: loginReturnUri
+
+loginReturnUri: After the Grab authorization endpoint successfully authenticates the request, this value can be fetched on your redirect page via GrabID.getLoginReturnURI. If not provided, loginReturnUri defaults to the page the login request originated from.
+
+#### Step 2 - Receive and process results of authorization request
+
+Once the authorization request has successfully completed, the user will be redirected to the redirectUri you specified when instantiating the GrabID client. This should be a dedicated [redirect page](./app/redirect.html). There are a couple important things you need to do here:
+
+```javascript
+let returnUri = GrabID.getLoginReturnURI();
+GrabID.handleImplicitFlowResponse()
+window.location.assign(returnUri)
+```
+
+First, you need to make sure that the authorization was successful. GrabID.handleImplicitFlowResponse will process all of the values that came back from the authorization request and ensure that the user is logged in and has provided the appropriate consent.
+
+Next, you need to redirect the user to the appropriate page/route on your site. GrabID.getLoginReturnURI will provide the value, and then you need to route the user appropriately. In the case of our example, we use a simple window.location.assign to redirect the user manually.
+
+## IMPORTANT
+
+Make sure you handle the negative case, too! If the user failed to authenticate or failed to provide consent, you will need to handle that here. Refer to the sample [redirect page](./app/redirect.html) to see an example of how to handle failed authentication requests.
+
+#### Step 3 - Perform authorized operations
+
+In the implicit flow, there's a little less work needed to make calls to an endpoint that requires authentication.
+
+```javascript
+let tokenResult = GrabID.getResult();
+  window.fetch("https://grab.api.com/grabid/v1/oauth2/userinfo", {
+    method: 'GET',
+    headers: {
+      'Accept-Content': 'application/json; charset=utf-8',
+      Authorization: "Bearer " + tokenResult.accessToken,
+    },
+    mode: 'cors',
+  })
+  .then((response) => { 
+    response.json().then((userInfo) => {
+      this.setState({
+        userName: userInfo.name,
+        userEmail: userInfo.email,
+      });
+    });
+  })
+```
+
+Note that this is very similar to the auhorization flow, but we did not have to make an additional call to get a token. That's because the implicit flow directly returns a token that you can use to make requests instead of an access code that you exchange for a token.
+
+## Which to use - Authorization Code Flow or Implicit Flow?
+
+Unless you have a specific use case that requires you to use the Implicit flow, we highly recommend using the Authorization code flow. It does require a little more code to function, but it is also much more secure. Because the user must exchange an authorization code for an access token, it gives another layer of security.
```

Comment:
What would be the use case that dictates an Implicit flow? A little more guidance to developers who are not versed with OAuth 2.0 flow?

Reply:
Talked to Ari about this. You're right that we need to add more information here. I'm planning on doing this in a subsequent PR once we've had a chance to have a conversation about this.